src/components/Card/Card.module.scss

```
@@ -0,0 +1,58 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 8px 12px;
+  width: 155px;
+  position: relative;
+  background: #FFFFFF;
+  box-shadow: 0px 1px 8px rgba(0, 0, 0, 0.12);
+  border-radius: 10px;
+}
+
+.card:hover {
+  opacity: 0.8;
+  background: #f3f3f3;
+}
+
+.card__image {
+  margin: 3px 22px 3px 22px;

```

```suggestion
  margin: 3px 22px;
```

некрит, но глаз цепляется